src/steps/10-FinancialDetails/IGCE/EstimatesDeveloped.spec.ts

```diff
@@ -29,7 +42,7 @@ describe("Testing EstimatesDeveloped Component", () => {
       selectedPriceComparison: "More"
     });
     const str = wrapper.vm.overUnderStr;
-    expect(str).toBe("overestimated");
+    expect(str).toBe("underestimated");
```

Comment:
New expectation does not match the test text "checks getter overUnderStr string 'overestimated'".

Reply:
Something's wrong because this test passes and so does the next one.